php/WP_CLI/Bootstrap/IncludeFallbackAutoloader.php

```diff
@@ -17,8 +17,7 @@ final class IncludeFallbackAutoloader extends AutoloaderStep {
 	/**
 	 * Get the autoloader paths to scan for an autoloader.
 	 *
-	 * @return string[]|false Array of strings with autoloader paths, or false
-	 *                        to skip.
+	 * @return string[] Array of autoloader paths, or an empty array if none are found.
```

Comment:
This looks correct. I don't see how it could ever return `false`.

Reply:
Thanks for feedback